gsa/src/web/pages/reports/alertactions.js

```diff
@@ -152,16 +150,15 @@ class AlertActions extends React.Component {
   }
 
   onAlertCreated(response) {
```

Comment:
Do you need the full information of response? I suppose only passing the alertId would be fine here.